runtime/js/99_main.js

```diff
@@ -203,6 +203,52 @@ delete Object.prototype.__proto__;
     );
   }
 
+  class Navigator {
+    constructor(key = null) {
+      if (key != webUtil.illegalConstructorKey) {
+        throw new TypeError("Illegal constructor.");
+      }
+    }
+  }
```

Comment:
This is not spec conformant. Use the same construction technique we use in WebGPU. `Navigator.length` will be wrong.

It boils down to constructing the class with `Object.create`, branding the class using a symbol, and then checking the symbol in each method.